bokehjs/src/coffee/common/ui_events.coffee

```diff
@@ -16,38 +16,42 @@ class UIEvents extends Backbone.Model
     hit_area = @get('hit_area')
 
     unless options and options.testing
```

Comment:
apologies I get confused by `unless`. Am I understanding this correctly that this line means that we call `_this._hammer_element hit_area[0]` if we're not testing?

If that's the case, then I'd propose that another way of handling this would be to stub out `_hammer_element` in the test rather than have this testing detail in the main code. 

If it's not the case, can you explain what's going on here :sheep: